app/components/Option/OptionSelect.js

```diff
@@ -22,40 +22,51 @@ export function OptionSelect({
   icon,
   title,
 }) {
+  const indicatorStyle =
+    inputType === SCREEN_TYPE_CHECKBOX
+      ? styles.indicatorCheck
+      : styles.indicatorRadio;
+
   return (
     <View style={wrapperStyle}>
       {isValidType && (
-        <View
-          style={[styles.indicator, isSelected && styles.indicatorSelected]}>
+        <View style={[indicatorStyle, isSelected && styles.indicatorSelected]}>
           {isSelected && inputType === SCREEN_TYPE_CHECKBOX && (
-            <SvgXml
-              height={INDICATOR_WIDTH * 0.5}
-              style={styles.indicatorIcon}
-              width={INDICATOR_WIDTH * 0.5}
-              xml={icon}
-            />
+            <SvgXml width={Spacing.medium} xml={icon} />
           )}
           {isSelected &&
             (inputType === SCREEN_TYPE_RADIO || inputType === SCREEN_TYPE_DATE)}
         </View>
       )}
-      <Typography use='body1' style={styles.title} testID='label'>
+      <Typography style={styles.title} testID='label'>
         {title}
       </Typography>
     </View>
   );
 }
 
-const styles = StyleSheet.create({
+const inputStyle = StyleSheet.create({
```

Comment:
What do you think about putting this in the existing stylesheet? Not sure we need two here. something thing:

```
const styles = StyleSheet.create({
  input: {
      alignItems: 'center',
      ...
   },
  indicatorRadio: {
    ...inputStyle.indicator,
    borderRadius: Spacing.medium * 2,
  },
```

Reply:
Sure thing, I can give that a shot.

Reply:
We're getting an error of `TypeError: undefined is not an object (evaluating styles.indicator)` when we try this. 

The reason it was separate initially was to avoid this error. Is there a way to extend a style within `styles`? In Sass it'd be akin to a mixin to coordinate shared properties.

Reply:
Scratch that, we wound up migrating it into `forms.ts`.